### ISSUE TITLE
Missing scene change behaviours

### ISSUE THREAD

**Issue by Helco** [Helco/zzio#181]
Postponed from #176 

- [ ] Handle actor head IK
- [ ] Save savegame
- [ ] Investigate updating active trigger without triggering at scene change
- [ ] Create elemental lock effects
- [ ] Create collection wizforms
- [ ] Show hot scene warning
- [ ] Show scene name notification

**Comment by AcipenserSturio** [comment]
Add "Amy running forward for about 1 second after changing scenes" to the list?

**Comment by Helco** [comment]
Yeah this is a bug although not related to scene changes, what happens is that I am not correctly calculating box-shaped triggers (causing them to be huge, this can be seen in the scene editor) and there are several of them in sc_1244 (maybe other swamp scenes as well) around the level edges deep into the swamp. So:

  1. Amy walks in, is placed outside and away from the actual entry trigger
  2. immediately one of the edge triggers activate and send Amy back
